shared/ui-components/src/forms/ProgressNav.tsx

```diff
@@ -1,5 +1,5 @@
 import React from "react"
-import { onClientSide } from "../helpers/nextjs"
+import { OnClientSide } from "../helpers/nextjs"
```

Comment:
Caps looks odd to me and doesn't match other hook/function names…was there a reason to change it?

Reply:
There is a lining error for that function, thats why I renamed it to capital case.

Reply:
Yeah, this is what spurred me to file #509, which I believe will be the real fix and hopefully supersede this function, but in the meantime I think it's better that we just get linting to pass. I'm also fine if you think readability would be better to keep it as-is and then just turn off the eslint rule in this case until we fix it?